contrib/ruby/ext/trilogy-ruby/cext.c

```diff
@@ -156,6 +156,10 @@ static void handle_trilogy_error(struct trilogy_ctx *ctx, int rc, const char *ms
         rb_raise(Trilogy_EOFError, "%" PRIsVALUE ": TRILOGY_CLOSED_CONNECTION", rbmsg);
     }
 
+    case TRILOGY_UNSUPPORTED: {
+        rb_raise(Trilogy_QueryError, "%" PRIsVALUE ": TRILOGY_UNSUPPORTED", rbmsg);
```

Comment:
QueryError doesn't seem quite right to me. Maybe:

```suggestion
        rb_raise(Trilogy_BaseConnectionError, "%" PRIsVALUE ": TRILOGY_UNSUPPORTED", rbmsg);
```

Reply:
I don't recall why I picked QueryError 🤔 I think I saw it in the output and assumed it was not able to be changed. Anyway, I've fixed this so I think the PR is good to go.